catalog/ddms/catalog-ddms-transformer/src/main/java/org/codice/alliance/ddms/types/RestrictionAttributes.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.ddms.types;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+public class RestrictionAttributes implements MetacardType {
+  private static final String NAME = "restrictions";
+  private static final Set<AttributeDescriptor> DESCRIPTORS;
+
+  private static final String RESTRICTION_NS = "ext.restrictions.";
+  public static final String COPYRIGHT_APPLIES = RESTRICTION_NS + "copyright-applies";
+  public static final String INTELLECTUAL_PROPERTY = RESTRICTION_NS + "intellectual-property";
+  public static final String PRIVACY_ACT = RESTRICTION_NS + "privacy-act";
```

Comment:
Since these are similar should we add `-applies` to all?

Reply:
Sure